src/manifest.py

```diff
@@ -406,6 +407,29 @@ def _update_manifest(self, path, datas, changes):
             log.info("Updating %s", path)
             self._dump_manifest(path)
 
+    def has_version_changed(self, selected_data) -> bool:
```

Comment:
This would be better be a method (or even a property) on `ExternalBase` objects.